### DISCUSSION TITLE
🆕 Welcome to the '🐙 Issues Feedback' category!

### DISCUSSION THREAD

**Discussion by azenMatt** [community/community#4253] [Projects and Issues]
This space is all about discussing with the community, including members of the GitHub product team, how we can improve the experience of planning and tracking development work on GitHub. We've added a couple of discussion topics to get your feedback on potential new features that are under consideration, but also want to hear from you about what would be most useful!

[Learn more about our vision for Issues](https://github.blog/2021-06-23-introducing-new-github-issues/) and [sign up for access to our latest features](https://github.com/features/issues).

We are hoping to foster productive and collaborative conversations, so please [check out the README](https://github.com/github/feedback#making-suggestions) for guidance on kicking off a successful discussion.

**Comment by ElijahLynn** [comment]
The UX of the "Task Lists! Give Feedback" showing on issues now was confusing, because it just took me to the discussions page with a Issues Feedback filter on it. I was expecting it to take us to an existing discussion. Just an FYI on the UX, it confused me. 

![image](https://user-images.githubusercontent.com/1504756/123143859-dcdcfc00-d40f-11eb-981e-d8c2c91d87de.png)

**Reply by jaredsb** [reply]
Thanks for the feedback @ElijahLynn! Feel free to submit a new discussion about your feedback to get the conversation started.

**Reply by SteOlivier** [reply]
I actaully reached this point same way @ElijahLynn just did. Came to give my feedback that the new task lists are in my opinion one of the best new improvements to github in the last year, but it seems that there is no place to give feedback.

Well take my opinion here anyway.

**Reply by Matzee** [reply]
ok

**Reply by shwq774791378** [reply]
نعم

**Reply by 229c9cf0** [reply]
Instead of pointing at the whole ["Issues Feedback" category](https://github.com/github/feedback/discussions/categories/issues-feedback), the link could at least point at ["Issues Feedback" filtered on the "Task List" label](https://github.com/github/feedback/discussions/categories/issues-feedback?discussions_q=category%3A%22Issues+Feedback%22+label%3A%22Task+Lists%22).

**Comment by Genius398** [comment]
h

**Reply by evanschisenga** [reply]
hey

**Comment by hafsabatool7** [comment]
Can you introduce intelligent task list? That is, have certain task list elements associated with separate files or folders. So that when the folder/file is changed, the respective task list appears but if a pull request does not contain changes in that specific folder, the respective task list should not appear.

**Comment by dt-woods** [comment]
Big fan of the task lists in GitHub Issues. One thing I'd like to see is the tasks progress bar on the Issues page even when there are >100 tasks in an issue. Unfortunately, when you have 101+ tasks, the progress bar is replaced with a task icon and 100+. Not even a popup progress on mouse hover. You have to click to open the Issue to see the progress, which is disappointing; I can't scroll through all open issues to see their progress. Otherwise, very excited about task lists.
